src/main/resources/swagger/api-docs.yaml

```diff
@@ -1379,13 +1664,13 @@ paths:
   /api/workspaces/{workspaceNamespace}/{workspaceName}/method_configs/{configNamespace}/{configName}/validate:
     get:
       responses:
-        '200':
+        200:
           description: Successful Request
           schema:
             $ref: '#/definitions/ValidatedMethodConfiguration'
-        '404':
+        404:
           description: Workspace or Method Configuration does not exist
-        '500':
+        500:
```

Comment:
I got rid of the empty description. There's a ton of 500s in here that don't reference `ErrorReport`; I'm tempted to just leave them. Do you think it's worth auditing all of them?